### ISSUE TITLE
Enter casename (shortcompsetname.grid.codeversion.somekindofinfo.yyyymmdd)

### ISSUE THREAD

**Issue by mvertens** [NorESMhub/noresm3_dev_simulations#56]
**Purpose:**
Duplicate of n1850.ne30_tn14.hybrid_fatessp.20241031 but with an updated twostream scheme for FATES.  
The only difference with n1850.ne30_tn14.hybrid_fatessp.20241031 is a change to user_nl_clm (see below).
The goal is to run 20 years.

___
**Description:**
- use of FATES-SP and cold start.
- NOTE: a memory leak is present in the run and hits agains the node `limit` when running past 11 years.
See https://github.com/NorESMhub/NorESM/issues/550

___
**Description:**

- **create_newcase commnad**
```
 ./create_newcase --case <user_case_path>/n1850.ne30_tn14.hybrid_fatessp.20241108 \\
 --compset 1850_CAM%DEV%LT%NORESM%CAMoslo_CLM60%FATES-SP_CICE_BLOM%ECO_MOSART_DGLC%NOEVOLVE_SWAV_SESP  \\
 --res ne30pg3_tn14 \\
 --project nn9039k --run-unsupported --mach betzy 
```

- **user_nl_cam**
```
 use_aerocom = .true.
 interpolate_nlat   = 192
 interpolate_nlon   = 288
 interpolate_output = .true.
 history_aerosol = .true. 
 zmconv_c0_lnd  =  0.0075D0
 zmconv_c0_ocn  =  0.0300D0
 zmconv_ke =  5.0E-6
 zmconv_ke_lnd =  1.0E-5
 clim_modal_aero_top_press =  1.D-4
 bndtvg = '/cluster/shared/noresm/inputdata/atm/cam/ggas/noaamisc.r8.nc'
```
- **user_nl_blom**
`ICFILE = "/cluster/shared/noresm/inputdata/restart/NOIIAJRAOC20TR_TL319_tn14_ppm_20240816/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_ppm_20240816.blom.r.1775-01-01-00000.nc"`

- **user_nl_cice**
`ice_ic = "/cluster/shared/noresm/inputdata/restart/NOIIAJRAOC20TR_TL319_tn14_ppm_20240816/rest/1775-01-01-00000/NOIIAJRAOC20TR_TL319_tn14_ppm_20240816.cice.r.1775-01-01-00000.nc"`

- **user_nl_clm**
Add the following to user_nl_clm
```
fates_paramfile='/cluster/work/users/rosief/git/NorESM_alpha07test/components/clm/src/fates/parameter_files/fates_params_twostream.nc
z0param_method='ZengWang2007'
```
- **xmlchange commands**
```
./xmlchange NTASKS=1920
./xmlchange NTASKS_OCN=256
./xmlchange ROOTPE=0
./xmlchange ROOTPE_OCN=1920
./xmlchange BLOM_VCOORD=cntiso_hybrid,BLOM_TURBULENT_CLOSURE=
./xmlchange STOP_OPTION=nyears
./xmlchange STOP_N=7
./xmlchange REST_N=1
./xmlchange REST_OPTION=nyears
./xmlchange RESUBMIT=3
./xmlchange --subgroup case.run JOB_WALLCLOCK_TIME=48:00:00
./xmlchange --subgroup case.st_archive JOB_WALLCLOCK_TIME=03:00:00
```
___
**Compset long name**
`1850_CAM%DEV%LT%NORESM%CAMoslo_CLM60%FATES-SP_CICE_BLOM%ECO_MOSART_DGLC%NOEVOLVE_SWAV_SESP`

___
**Model grid long name**
`a%ne30np4.pg3_l%ne30pg4.pg3_oi%tnx1v4_r%r05_w%null_z%null_g%gris4_m%tnx1v4`

___
**Case directory:**
- Locally on betzy: 
   `<user_pathname>/n1850.ne30_tn14.hybrid_fatessp.202401031`
- On github: 
 
___
**Code version on github:**
- https://github.com/NorESMhub/NorESM/releases/tag/noresm2_5_alpha07
- CAM needs to be updated - modify Externals.cfg to have the following change:
```
[cam]
protocol = git
tag = noresm2_5_015_cam6_3_158.  
repo_url = https://github.com/NorESMhub/CAM
local_path = components/cam
externals = Externals_CAM.cfg
required = True
```

___
**Diagnostics:**

- Atmosphere:

- Ocean / sea ice / hamocc: 
___
**Output:**
- Raw output: ` /nird/datalake/NS9560K/noresm3/cases/n1850.ne30_tn14.hybrid_fatessp.20241108`
___
**Contacts:**
@mvertens, @kjetilaas 
___
**Extra details:**

**Comment by adagj** [comment]
Hi, Is this simulation running? Do we meet on Thursday or is that too early?

**Comment by kjetilaas** [comment]
Yes, this is running, but I didn't manage to start it before Monday so it will not be ready before tomorrow morning. It has completed about 11 years, so it will probably finish late Thursday or Friday. 

If it is only this simulation, we might skip the meeting tomorrow. But if we think it would be useful to meet anyway, we could make some plot from the first 10 years or so. 

What do you think? @rosiealice @mvertens @adagj

**Comment by adagj** [comment]
Let's meet tomorrow :-)

**Comment by kjetilaas** [comment]
@adagj, now the rest of the simulation is finished and archived (same folder). 

@mvertens, I have uploaded the case information with CaptureCase. I hope I did it right?
